### DISCUSSION TITLE
dev build instructions or npm dev package

### DISCUSSION THREAD

**Discussion by doczoidberg** [handsontable/hyperformula#508] [General]
One can build the sources from dev branch as described here: https://handsontable.github.io/hyperformula/guide/building.html#build-the-project

but it's not possible to use the created library. At least in an angular project.
```

Error: Language not registered.
Error: Language not registered.
    at Function.getLanguage (HyperFormula.js:3553)
    at new Config (Config.js:154)
```

It would be nice to have further documentation on how to use the compiled dev branch or even better would be a npm dev build which continously updates.

**Comment by wojciechczerniak** [comment]
The documentation is focused on HyperFormula, what you need to do after building the project is a package management responsibility i.e. https://medium.com/@the1mills/how-to-test-your-npm-module-without-publishing-it-every-5-minutes-1c4cb4b369be 

I like the idea of Continous Delivery, but it is not a common practice for NPM packages. We're releasing `beta` versions sometimes, but release cycles are prefered and stable. Developers interested in `develop` build can use the techniques mentioned in the article.

Our @handsontable/support team is checking if there is an issue with Angular.

**Comment by doczoidberg** [comment]
following steps worked for me in my angular app:

1. npm run bundle-all (for the /develop branch)
2. npm pack
It will install a package (tarball) as a file on your device.
3. Install the package from the file https://docs.npmjs.com/cli/install 
4. Import it to your angular application.
